Makefile

```
@@ -11,6 +11,20 @@ GOPATH := ${shell go env GOPATH}
 GOOS := ${shell go env GOOS}
 GOARCH := ${shell go env GOARCH}
 
+SOURCE_GIT_TAG ?=$(shell git describe --long --tags --abbrev=7 --match 'v[0-9]*' || echo 'v0.0.0-unknown-$(SOURCE_GIT_COMMIT)')
```

```suggestion
SOURCE_GIT_LATEST_TAG ?= $(shell git describe --tags `git rev-list --tags --max-count=1`)
SOURCE_GIT_TAG ?=$(shell git describe --long --tags --abbrev=7 --match 'v[0-9]*' || echo 'v0.0.0-unknown-$(SOURCE_GIT_COMMIT)')
```
we add a new `SOURCE_GIT_LATEST_TAG` which will be used in `build-krew`